docs/user-guide/projects/project-lists/README.md

```diff
@@ -13,12 +13,15 @@ In OpenProject you can create projects to collaborate with your team members, tr
 
 Your projects can be available publicly or internally. OpenProject does not limit the number of projects, neither in the Community edition nor in the Enterprise cloud or in Enterprise on-premises edition.
 
-| Topic                                                       | Content                                                      |
-| ----------------------------------------------------------- | ------------------------------------------------------------ |
-| [Select project lists](#select-project-lists)               | Get an overview of all your projects in the project lists.   |
-| [Filter project lists](#project-lists-filters)              | Adjust filters in the project lists.                         |
-| [Export project list](#export-project-lists)                | You can export the project list to XLS or CSV.               |
-| [Project overarching reports](#project-overarching-reports) | How to create project overarching reports across multiple projects. |
+| Topic                                                        | Content                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Select project lists](#select-project-lists)                | Get an overview of all your projects in the project lists.   |
+| [Filter project lists](#project-lists-filters)               | Adjust filters in the project lists.                         |
+| [Favorite project lists](#favorite-project-lists)            | Mark project lists as favorite.                              |
+| [Share project lists](#share-project-lists-with-users-and-groups-enterprise-add-on) | Share project lists with users and groups.                   |
+| [Public project lists](#share-with-everyone-at-openproject)  | Share project lists with everyone at OpenProject.            |
```

Comment:
Hi @MayaBerd, we have been using the term share with everyone in your instance for that instead of public lists to avoid the confusion between sharing inside of an instance and making it public even outside of your instance (users that are not even logged in).

Reply:
As we discussed also in chat, I think we should get rid of the term public and start using "Share with everyone" or something similar, and I would also change the explanation to "Share project lists with everyone at your instance."